gen/templates/factory/table/02_build.go.tpl

```diff
@@ -101,6 +101,17 @@ func (o {{$tAlias.UpSingular}}Template) Build() *models.{{$tAlias.UpSingular}} {
 	return m
 }
 
+func (o *{{$tAlias.UpSingular}}Template) From(locator func() models.{{$tAlias.UpSingular}}) {
```

Comment:
1. Why is `locator` a function instead of the model?
2. I think this method should not be on the template, but should be a method of the factory directly and will construct and return a new template similar to `f.New<ModelName>`.   
This makes it unlikely that the templated values will be different from the model itself

Reply:
1: for example, in the example I used (findRandomUser), if we create multiple models with `CreateMany`, wouldn't it make sense to use different random users? This is why I went with the function, but for any other use case I think of, a model would be fine. I can change it to that if you think it is better.

(there's a separate issue currently that CreateMany only creates relations for the first returned model, it's empty for all others - but assuming that's fixed)

Reply:
Updated: method removed, and the other methods now accept a model directly.